scipy/stats/contingency.py

```diff
@@ -271,3 +272,123 @@ def chi2_contingency(observed, correction=True, lambda_=None):
+
+    Notes
+    ------
+    Cramer's V, Tschuprow's T and Pearson's Contingency Coefficient, all measure
+    the degree to which two nominal or ordinal variables are related, or the
+    level of their association. This differs from correlation, although many
+    often mistakenly consider them equivalent. Correlation measures in what way
+    two variables are related, whereas, association measures how related the
+    variables are. As such, association does not subsume independent variables,
+    and is rather a test of independence. A value of 1.0 indicates perfect
+    association, and 0.0 means the variables have no association.
+
+    Both the Cramer's V and Tschuprow's T are extensions of the phi coefficient.
+    Moreover, due to the close relationship between the Cramer's V and
+    Tschuprow's T the returned values can often be similar or even equivalent.
+    They are likely to diverge more as the array shape diverges from a 2x2.
+
+    Lastly, all of the metrics are subject to the introduction of bias from
+    increased counts or increased number of groups among observed values.
+    In other words, as each value of N approaches infinity the association
+    statistic will approach 1.0.
+
+
+    References
+    ----------
+    .. [1] "Tschuprow's T",
+           https://en.wikipedia.org/wiki/Tschuprow's_T
+    .. [2] Tschuprow, A. A. (1939)
+           Principles of the Mathematical Theory of Correlation;
+           translated by M. Kantorowitsch. W. Hodge & Co.
+    .. [3] "Cramer's V", https://en.wikipedia.org/wiki/Cramer's_V
+    .. [4] "Nominal Association: Phi and Cramer's V",
+           http://www.people.vcu.edu/~pdattalo/702SuppRead/MeasAssoc/NominalAssoc.html
+    .. [5] Gingrich, Paul, "Association Between Variables",
+           http://uregina.ca/~gingrich/ch11a.pdf
+
+    Examples
+    --------
+    An example with 4x2 contingency table:
+
+    >>> from scipy.stats.contingency import association
+    >>> obs4x2 = np.array([[100, 150], [203, 322], [420, 700], [320, 210]])
+
+    Pearson's contingency coefficient
+    
+    >>> association(obs4x2, method="pearson")
+    0.18303
+
+    Cramer's V
+
+    >>> association(obs4x2, method="cramer")
+    0.18617
+
+    Tschuprow's T
+
+    >>> association(obs4x2, method="tschuprow")
+    0.14146
+    """
+    try:
+        arr = np.asarray(observed, dtype=int)
+    except ValueError as err:
+        raise ValueError("All array values must be float or int")
+
+    if len(arr.shape) != 2:
+        raise ValueError("method only accepts 2d arrays")
+    elif any([(True if type(a) == list else False) for a in arr]):
+        raise ValueError("All arrays must be non-empty and have "
+                         "equivalent length.")
+    try:
+        chi2_stat = chi2_contingency(arr, correction=correction,
+                                     lambda_=lambda_)
+    except Exception as err:
+        raise Exception(err)
+    else:
+        phi2 = float(chi2_stat[0]) / arr.sum()
```

Comment:
This use of `float` is not necessary.
```suggestion
        phi2 = chi2_stat[0] / arr.sum()
```